src/nodes/EnvironmentalEffects/Background.js

```diff
@@ -41,65 +41,65 @@ x3dom.registerNodeType(
             this.addField_MFString( ctx, "backUrl", [] );
 
             /**
-             *
+             * Image shown at the bottom of the background cube
              * @var {x3dom.fields.MFString} bottomUrl
              * @memberof x3dom.nodeTypes.Background
              * @initvalue []
              * @range [URI]
-             * @field x3dom
+             * @field x3d
              * @instance
              */
             this.addField_MFString( ctx, "bottomUrl", [] );
 
             /**
-             *
+             * Image shown at the front of the background cube
              * @var {x3dom.fields.MFString} frontUrl
              * @memberof x3dom.nodeTypes.Background
              * @initvalue []
              * @range [URI]
-             * @field x3dom
+             * @field x3d
              * @instance
              */
             this.addField_MFString( ctx, "frontUrl", [] );
 
             /**
-             *
+             * Image shown at the left of the background cube
              * @var {x3dom.fields.MFString} leftUrl
              * @memberof x3dom.nodeTypes.Background
              * @initvalue []
              * @range [URI]
-             * @field x3dom
+             * @field x3d
              * @instance
              */
             this.addField_MFString( ctx, "leftUrl", [] );
 
             /**
-             *
+             * Image shown at the right of the background cube
              * @var {x3dom.fields.MFString} rightUrl
              * @memberof x3dom.nodeTypes.Background
              * @initvalue []
              * @range [URI]
-             * @field x3dom
+             * @field x3d
              * @instance
              */
             this.addField_MFString( ctx, "rightUrl", [] );
 
             /**
-             *
+             * Image shown at the top of the background cube
              * @var {x3dom.fields.MFString} topUrl
              * @memberof x3dom.nodeTypes.Background
              * @initvalue []
              * @range [URI]
-             * @field x3dom
+             * @field x3d
              * @instance
              */
             this.addField_MFString( ctx, "topUrl", [] );
 
             /**
-             *
+             * Scale images of the background cube (### verification needed as I didn't found any example running trought the corresponding coding)
```

Comment:
I still do not figured out in current version of x3dom how to scale (or not to scale) the background image cube in a way that these pics are not zoomed-in.

Reply:
The zoom is unrelated to scale. The zooming comes from a change in the size of the sphere used as the surface for the background texture, from 10000 to 1000. I think 1000 worked better for VR. For more control it is preferable to just use a big sphere in the scene instead of background.

Reply:
I could imagine that it would be better to see this constant 1000 at the caller within `Context.prototype.setupScene`.
Maybe it even would be better to add an optional `radius` attribute to node `Background`.

Reply:
Yes, I think that this is a good idea. The current setup is really obscure.